src/repr/src/row.rs

```diff
@@ -1193,6 +1350,172 @@ impl RowPacker<'_> {
         })
     }
 
+    /// Pushes a `DatumRange` that is considered empty.
+    ///
+    /// - To construct other ranges, use [`Self::push_range`],
+    ///   [`Self::push_range_with`].
+    pub fn push_empty_range(&mut self) {
+        self.row.data.push(Tag::Range.into());
+        // Untagged bytes only contains the `RANGE_EMPTY` flag value.
+        self.row.data.extend_from_slice(&1u64.to_le_bytes());
```

Comment:
Can this use `RANGE_EMPTY` directly instead of hard coding 1?

Reply:
This is hardcoding the size of the untagged bytes' length as 1; I can make this clearer using `push_untagged_bytes`.

Reply:
Note that I plan to move this away from the untagged bytes style encoding, which obviates the need to do this in the first place.